device.go

```diff
@@ -45,6 +45,10 @@ type DeviceDesc struct {
 
 	// Configuration information
 	Configs map[int]ConfigDesc
+
+	iSerialNumber       int   // The index into the GetStringDescriptor() to get the Serial Number
+	iProduct            int   // The index into the GetStringDescriptor() to get the Product name
+	iManufacturer       int   // The index into the GetStringDescriptor() to get the Manufacturer name
```

Comment:
actually, on second thought, putting them directly in Configuration and Interface will mean a config needs to be claimed, which should not be required for getting a descriptor.

So perhaps something like this:
```
Device.ConfigDescription(cfg int)
Device.InterfaceDescription(cfg, intf, alt int)
```